satellite/console/consoleweb/server.go

```diff
@@ -533,3 +528,12 @@ func (server *Server) initializeTemplates() (err error) {
 
 	return nil
 }
+
+// stripPort strips the port from hostport string if any.
+func stripPort(hostport string) string {
+	i := strings.Index(hostport, ":")
```

Comment:
This won't do the right thing with ipv6 addresses, maybe use `url.Parse` instead?

Reply:
don't work as expected(

Reply:
Wait, do we even need to remove the `:`?

Reply:
yes, because we use `host` on frontend side to create unique `tokenKey` cookie

Reply:
Yes, I mean. Let's say we have the same site on two different ports, wouldn't you want to use two different keys?

Reply:
no, I guess. So you suggest to add port to domain for token identifier?

Reply:
But it seems that `request` doesn't contain port if we use `dns` resolving, where port could be one of `80 or 443`

Reply:
It is not a problem if we have same site on different ports, we can use the same key. But if we have 2 different satellites under same domain but different ports, than yes - we need different keys

Reply:
Also is there a reason we cannot just use `_tokenKey`?

Reply:
`us-east.bla.bla`
`asia.bla.bla`
are different satellites

but wait it's redundant I guess as we store cookies by separate host keys in browser by default

Reply:
Yeah, that's kind of what I was thinking, that you can set secure/samesite etc. flags... that should contain it within the same domain. So the additional host shouldn't be necessary. https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Write_a_new_cookie